tests/rmi.bats

```diff
@@ -27,6 +27,15 @@ load helpers
   [ "$output" == "" ]
 }
 
+@test "remove multiple images errors" {
```

Comment:
suggest changing "images" to "non-existent images"

Reply:
updated， PTAL.